microsoft-edge/webview2/code-samples-links.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Links to WebView2 sample code hosted on GitHub.
+title: WebView2 sample code
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/18/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, source code, sample code, samples, code samples, WebView sample code, WebView2 sample code
+---
+# WebView2 sample code
+
+The following WebView2 samples are available in the Get Started guides for WebView2 and in the [WebView2 Samples repo](https://github.com/MicrosoftEdge/WebView2Samples).
+
+
+<!-- ====================================================================== -->
+## Sample code for Get Started guides
+
+The WebView2 "Get started" guides use starter code for Win32, Windows Presentation Foundation (WPF), WinForms, and WinUI.
+
+| Task | Article | Code |
+|---|---|---|
```

Comment:
In column 1, only put the platform name.  The article title column is already verbose.  This will make skimmable.